.github/workflows/windows.yml

```
@@ -0,0 +1,75 @@
+name: Windows CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+## events but only for the master branch
+on:
+  push:
+    branches:
+      - 'develop'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  tests:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: cache
+        id: cache-target
+        uses: actions/cache@v2
+        with:
+          path: c:/users/runneradmin/appdata/local/pip/cache
+          key: ${{ runner.os }}-${{ matrix.python-version }}
```

That hard-coded path is not the same on Github. Try:
```suggestion
      - name: Get pip cache dir
        run: echo "pip_cache_dir=$(pip cache dir)" >> $GITHUB_ENV
        shell: bash
        
      - name: cache
        id: cache-target
        uses: actions/cache@v2
        with:
          path: ${{ env.pip_cache_dir }}
          key: ${{ runner.os }}-${{ matrix.python-version }}
```

---

Ahh I see you've just looked them up.